yaksh/test_models.py

```diff
@@ -204,6 +207,29 @@ def test_get_status(self):
         # Then
         self.assertEqual(status, module_status)
 
+        # Module in progress
+
+        # Given
+        self.course_status.completed_units.add(self.learning_unit_one)
```

Comment:
The test case name is a bit confusing.` test_get_course_status` is more apt, IMHO.

Reply:
changed the name to `test_get_module_status` since module status is being tested.